### DISCUSSION TITLE
ranks.yml "MonsterKills"

### DISCUSSION THREAD

**Discussion by Nesseley** [Zrips/CMI#2034] [Q&A] [answered]
**Question: **
Can I make it so it checks for more than one mob?
- "MonsterKills:cow:500" I want to check kills for both cow and sheep. And add them together. Is this possible?

Example: - "MonsterKills:cow,sheep:500"

**Comment by CloudeLecaw** [comment, marked as answer]
No that not possible
you can make checks for cow and sheep and have both as requirments like 10 cows and 20 sheeps needed to be killed.
but you can add them like 10 cows/sheeps  must be killed.
but you could ask for animals killed... but this would not only inclue cows and sheeps...

you know you can file them here https://www.zrips.net/cmi/rankup-system/

if you wish to have something like **_- "MonsterKills:cow,sheep:500"_**
then make a feature request and descripe what you wish it could do and maybe some extanted informations or why this feature would been usefull

This would need new stuff to added, currently the stats of MC do not calculate them together.
